forms/HtmlEditorField.php

```diff
@@ -323,8 +323,12 @@ public function MediaForm() {
 		$fileField->setAttribute('data-multiselect', true);
 		$columns = $fileField->getConfig()->getComponentByType('GridFieldDataColumns');
 		$columns->setDisplayFields(array(
-			'CMSThumbnail' => false,
-			'Name' => _t('File.Name'),
+			'StripThumbnail' => false,
+			'Title' => _t('File.Title'),
+			'Created' => singleton('File')->fieldLabel('Created'),
+		));
+		$columns->setFieldCasting(array(
+			'Created' => 'Date->Nice'
```

Comment:
For me the time it was created/uploaded is not very useful and just adds clutter - but appreciate it could be different for other users. The sort will still take the time in to account I believe because the sort is done on the original rather than presented value, it's just not shown here. This is consistent with presentation of created date in the Files area currently. @clarkepaul do you have any thoughts on this one?